src/content/1.7/development/internationalization/add-a-language.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Add a language
+aliases:
+  - /1.7/development/add_a_language
+  - /1.7/development/add-a-language
+---
+
+# Add a language
+
+Before adding a language in Prestashop, you must check if the language is available in [Crowdin](https://crowdin.com/project/prestashop-official).
```

Comment:
We should explain what to do if the language cannot be found on crowdin. Maybe redirect to Crowdin and say "you must contact prestashop content team to add this language to the list" ?

Reply:
I ask to Louise

Reply:
:ok_hand:

Reply:
:ok_hand: